File: js/bubbleSort.test.js
const { bubbleSort } = require('./bubbleSort.js')

test('do you even sort, bro?', () => {
  let res = bubbleSort([1, 0, 2, 3, 4, 5])
  expected = [0, 1, 2, 3, 4, 5]
  expect(res).toEqual(expected)
})

File: python/bubble_sort_test.py
"""
test cases for bubble sort
"""
from bubble_sort import bubble_sort

def test_do_you_even_sort():
    response = bubble_sort([1, 0, 2, 3, 4, 5])
    expected = [0, 1, 2, 3, 4, 5]
    assert response == expected
